plugin-mssql/pom.xml

```diff
@@ -39,8 +39,8 @@
             <artifactId>plugin-jdbc</artifactId>
         </dependency>
         <dependency>
-            <groupId>net.sourceforge.jtds</groupId>
-            <artifactId>jtds</artifactId>
+            <groupId>com.microsoft.sqlserver</groupId>
```

Comment:
The use of a new driver causes some backwards-compatibility breaking changes, i.e. already created robots that use this plugin will break with a new release. I've asked the author of the support ticket and Sjoerd whether this is acceptable or not. 

We can mediate some of the problems with the following:
Override `buildConnection(...)` in `MSSQLConnectionStringFactory` to perform additional logic on the `connectionString` parameter. Here we can filter `jtds` from the protocol-part of the string. And we can convert the path-part of the string (the database) to a property.

The examples should still be updated to support the new case instead of the JTDS one.

See my latest commit for my suggestion, please revert it if you're not happy with it. I have not validated whether this works for both old and new connection strings (can't even run the tests locally, so hopefully the build passes).

The only backward-compatibility breaking change for adopting a new driver _might_ be the properties used, but I'm not certain. For example, the way to denote `instances` might differ.

Reply:
Great stuff ! The suggestions seems to work in most cases. It appears that the driver changes remains to be a breaking change, because (really) old versions of MSSQL are not supported in this MS driver*.

Thus I would like to put up for consideration a big version leap (e.g 4.0.0) for this change and going back to version branches (4.x.x  and 3.6.x) and maintain 3.6 as a legacy supported version. 

_*) For example our demo livelink setup does not work with this driver._
`SQL Server version 8 is not supported by this driver. `